boards/x230-flash/x230-flash.config

```diff
@@ -16,7 +16,7 @@ CONFIG_LINUX_USB=y
 CONFIG_LINUX_E1000E=y
 
 export CONFIG_USB_BOOT_DEV="/dev/sdb1"
-export CONFIG_BOOTSCRIPT=/bin/x230-flash.init
+export CONFIG_BOOTSCRIPT=/bin/install.init
 
```

Comment:
"updating" and "x230-flash.config" seems to be wrong in the commit message.

Reply:
oh sorry, I was reading it wrong. the commit message is fine :)

Reply:
@SebastianMcMillan This part would still be nice.